clustered/ehcache-clustered/src/assemble/docker/server/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM azul/zulu-openjdk-alpine:17-jre-latest
```

Comment:
Should this be a reference to an immutable tag?

Reply:
I've asked myself the question and I think that's better to use lastest. Otherwise we are forced to put a process in place to handle the update. Latest also does not harm for a sample file or something deployed OSS I think. Anybody not happy with that could grab the code and rebuild.

Reply:
Note: my answer would be different when we can add later a process to generate and deploy docker image to Docker hub ;-) for now for a sample in the kit i don't see any issue.